packages/admin-client/src/components/mappings/CreateMappingForm.tsx

```diff
@@ -36,113 +38,120 @@ const CreateMappingForm = ({ open, onClose }: CreateMappingFormProps) => {
     apiKey: '',
   });
 
-
   const handleSubmit = async (e: React.FormEvent) => {
     e.preventDefault();
 
-    const langFlowResponse = await langflowMappingControllerCreate({
-      body: {
-        model: formData.modelName,
-        url: formData.url,
-        historyComponentID: formData.historyComponentID
+    try {
+      const liteLLMResponse = await liteLlmControllerCreate({
+        body: {
+          model_name: formData.modelName,
+          litellm_params: {
+            model: formData.modelName,
+            api_base: formData.url,
+            api_key: formData.apiKey,
+            custom_llm_provider: formData.provider
+          }
+        }
+      });
+
+      if (liteLLMResponse.error) {
```

Comment:
The try throw then use error approach is pretty messy, I understand if you don't want to fix it now, but we should later